packages/create-snap/package.json

```
@@ -0,0 +1,92 @@
+{
+  "name": "@metamask/create-snap",
+  "version": "0.32.1",
+  "description": "A CLI for creating MetaMask Snaps.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/snaps-monorepo.git"
+  },
+  "license": "ISC",
+  "bin": {
+    "create-metamask-snap": "./dist/main.js"
```

Should this be `create-snap`?

```suggestion
    "create-snap": "./dist/main.js"
```